website/orders/templates/orders/order_list.html

```diff
@@ -2,6 +2,27 @@
 
 <div class="container-sm" id="item-orders-{{ shift.id }}">
     <link rel="stylesheet" href="{% static 'orders/css/item-list.css' %}"/>
+
+    <!-- Order buttons -->
+    {% if show_order_buttons %}
+        <div v-if="!shift.finalized && shift.can_order && shift.is_active" class="row row-cols-2 row-cols-md-4">
+            <div class="col mb-3" v-for="product in products">
+                <div class="p-1 d-flex flex-column h-100 rounded" style="background-color: var(--background-shade);">
+                    <p class="m-0 order-name small"><% product.name %></p>
+                    <p class="m-0 item-price small">€<% (Math.round(product.current_price * 100) / 100).toFixed(2) %></p>
```

Comment:
What is this? What's wrong with decimal fields?

Reply:
The price is displayed with the least amounts of digits possible, this makes sure that all prices are displayed as they should.

Example: `0,60` is displayed as `0.6` in the API.